vllm/core/scheduler.py

```diff
@@ -294,23 +300,27 @@ def create_empty(cls) -> "SchedulerPrefillOutputs":
 
 
 def seq_group_metadata_builder():
-    return SequenceGroupMetadata(request_id="",
-                                 is_prompt=False,
-                                 seq_data={},
-                                 sampling_params=None,
-                                 block_tables={})
+    return SequenceGroupMetadata(
+        request_id="",
+        is_prompt=False,
+        seq_data={},
+        sampling_params=None,
+        block_tables={},
+    )
```

Comment:
Seems like formatting noise? If this was not done by the vLLM pre-commit, can you revert it to minimize the change in this file?

Reply:
Yup, not sure how this got in but when it's reverted pre-commit doesn't complain